group-coordinator/src/main/resources/common/message/ShareGroupMemberMetadataKey.json

```diff
@@ -0,0 +1,28 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// KIP-932 is in development. This schema is subject to non-backwards-compatible changes.
+{
+  "type": "data",
+  "name": "ShareGroupMemberMetadataKey",
+  "validVersions": "10",
```

Comment:
Though we have keys defined till `9` currently but the KIP defines this key at version `10`. SHortly there will be a key defined by version `9` as well, as part of KIP-932 itself.

Reply:
I would expect `ShareGroupMetadataKey` and `ShareGroupMetadataValue` in the KIP to be defined at the same time. I'd prefer not to leave gaps in the versions, so if we are introducing a subset of the KIP's record schemas, I suggest renumbering them in the KIP so we get a continuous sequence once the PR is merged.

Reply:
Done.